Makefile

```diff
@@ -10,7 +10,7 @@ all: install-tools fetch-spec process-spec generate-client test
 
 process-spec:
 	openapi-down-convert --input spec/openapi.json --output spec/openapi-3.0.json
-	sed -i '' 's/"type": "null"/"nullable": true/g' spec/openapi-3.0.json
+	sed -i 's/"type": "null"/"nullable": true/g' spec/openapi-3.0.json
```

Comment:
The `-i ''` flag was causing me issues on Linux. Not sure if the Darwin sed is compatible with `-i` instead of `-i ''` but I don't have the hardware to test easily.

Reply:
Yeah FreeBSD Sed is not, I should just install gnused